graph/resolvers/podcast.js

```diff
@@ -12,15 +14,31 @@ export default {
       .first()
       .then((podcast) => podcast)
   },
+
+  bestPodcasts:async(_,{page})=>{
+    return convertBestPodcastsResponse(await getBestPodcasts(page))
+  },
+
+
+  podcastFromListenNote:async(_,{id})=>{
+    return convertGetPodcastResponse(await getPodcast(id))
+  },
+
+  searchPodcasts:async(_,{query,offset})=>{
+    return convertSearchPodcastsResponse(await search(query,offset))
```

Comment:
Just to clarify, so whenever users are searching a podcast (or if the app wants to retrieve additional information from a specific podcast) this query is directly relayed to the ListenNotes API right?
(So no interaction happens with our database?)

Reply:
yup , because our database is not huge , so anyway it will end up resorting to listennote api